test/interceptor/download_file_interceptor_test.dart

```diff
@@ -0,0 +1,118 @@
+import 'package:dio/dio.dart';
+import 'package:fluffychat/data/network/exception/dio_duplicate_download_exception.dart';
+import 'package:fluffychat/data/network/interceptor/download_file_interceptor.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/annotations.dart';
+import 'package:mockito/mockito.dart';
+import 'package:matrix/matrix.dart';
+
+import 'download_file_interceptor_test.mocks.dart';
+
+@GenerateMocks(
+  [
+    Dio,
+    Logs,
+    RequestInterceptorHandler,
+    ErrorInterceptorHandler,
+    ResponseInterceptorHandler,
+  ],
+)
+void main() {
```

Comment:
add cases:
- no item in `DownloadFileInterceptor`: verify after request: handler, size, ...
- one item existed, add new item (not duplicated): verify ...
- 2 item in downloading, 1 item will be success: verify new size of map, verify handler
- 2 item in downloading, 1 item will be error: verify new size of map, verify handler

Reply:
done